docs/installation.rst

```diff
@@ -71,7 +71,7 @@ If you are on Fedora, install the build dependencies with this command:
 
 .. code-block:: bash
 
-   sudo dnf install -y gcc make cairo-devel cairo-gobject-devel cmake bzip2-devel expat-devel file-devel glib2-devel gobject-introspection-devel libcurl-devel libmodulemd-devel libxml2-devel openssl-devel python3-devel rpm-devel sqlite-devel xz-devel zchunk-devel zlib-devel
```

Comment:
I believe we still need this to be able to build createrepo_c from source, we can't drop it until we are able to use createrepo_c as a binary package.

Reply:
unfortunately true